README.md

```diff
@@ -4,8 +4,7 @@
 
 > Personal portfolio website
 
-![127 0 0 1_5500_index html](https://user-images.githubusercontent.com/77843050/143435530-94e24550-9803-46fd-928e-434ba433febb.png)
-
+![127 0 0 1_5500_index html(iPhone 6_7_8)](https://user-images.githubusercontent.com/77843050/143677066-188e9795-e5bc-489a-be2a-48f29f32abbb.png)
```

Comment:
## [OPTIONAL]
- [ ] Kindly, I think it would be nice if you put only the screenshot of the first section in the README section rather than the full screenshot of the project. This will prevent visitors from infinite scrolling in your README.

Reply:
Thank you Anderson for the feedback, I obviously have noticed that  I did not put this part as requested, I will come back to you later with changes made. 
thanks.